source/_docs/terminus/install.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Terminus Install
+terminusinstall: true
+terminuspage: true
+type: terminuspage
+layout: terminuspage
+permalink: docs/terminus/:basename/
+---
+<!-- Tab Nav -->
+<ul class="nav nav-tabs" role="tablist">
+  <li id="mactab" role="presentation" class="active"><a href="#mac" aria-controls="mac" role="tab" data-toggle="tab">Mac OS X</a></li>
+  <li id="linuxtab" role="presentation"><a href="#linux" aria-controls="linux" role="tab" data-toggle="tab">Linux</a></li>
+  <li id="wintab" role="presentation"><a href="#win" aria-controls="win" role="tab" data-toggle="tab">Windows</a></li>
+</ul>
+
+<!-- Tab Panes -->
+<div class="tab-content">
+  <!-- Unix: Mac Instructions -->
+  <div role="tabpanel" class="tab-pane active" id="mac">
+    <h2 class="tab-content-heading">Install Terminus for Mac OS X</h2>
+    <h3>Requirements</h3>
+    <ul>
+      <li>PHP Version 5.5.9 or later</li>
+      <li><a href="http://www.php-cli.com/">PHP-CLI</a></li>
+      <li><a href="https://getcomposer.org/download">Composer</a></li>
+    </ul>
+    <p class="instruction">Run the following command to install Terminus Alpha (<code>terminus1x</code>) with <a href="https://getcomposer.org/doc/00-intro.md">Composer</a>:</p>
+    <div class="copy-snippet">
+      <button class="btn btn-default btn-clippy" data-clipboard-target="#mac-install-composer"><img class="clippy" src="/source/docs/assets/images/clippy.svg" width="17" alt="Copy to clipboard"></button>
+      <figure><pre id="mac-install-composer"><code class="bash" data-lang="bash">cd $HOME ; mkdir terminus1x ; cd terminus1x ; echo '{"minimum-stability": "dev", "prefer-stable": true}' > composer.json ; composer require pantheon-systems/terminus dev-master ; echo 'alias terminus1x=$HOME/terminus1x/vendor/bin/terminus'>>$HOME/.bash_profile ; source $HOME/.bash_profile</code></pre></figure>
```

Comment:
Maybe we should tag `1.0.0-beta1` or something on Terminus, and remove `dev-master` from the `composer require`.